internal/model/to_translator.go

```diff
@@ -16,17 +16,23 @@ package model
 
 import "go.opentelemetry.io/collector/consumer/pdata"
 
+// ToMetricsTranslator interface to translate pdata.Metrics to protocol-specific data model.
```

Comment:
ToMetricsTranslator is an interface...

Reply:
This interface names could have been just "MetricsTranslator".

Reply:
Done. Name cannot be changed because we have a FromMetricsTranslator as well.